template/docroot/sites/default/settings/files.settings.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * @file
+ * Contains Files configuration.
+ */
+if ($is_acsf) {
+  $settings['file_private_path'] = "/mnt/gfs/$ah_group.$ah_env/files-private/$acsf_db_name";
+}
+else {
+  $settings['file_private_path'] = "/mnt/files/$ah_group.$ah_env/files-private";
```

Comment:
> Or I guess we could just treat every site as multisite. I don't know if there's any harm in that.

I feel this is the right way to go. 

If we can avoid using Acquia specific variables like `$ah_group.$ah_env` and use Drupal API functions like `conf_path()` instead this code will be a lot more portable and work on local, CI, as well as Acquia.